vmm/src/vm_config.rs

```diff
@@ -187,6 +187,12 @@ pub struct RateLimiterGroupConfig {
     pub rate_limiter_config: RateLimiterConfig,
 }
 
+#[derive(Clone, Debug, PartialEq, Eq, Deserialize, Serialize)]
+pub struct VirtioBlkAffinity {
```

Comment:
Even though this PR only adds support for virtio-blk queue pinning. It might make sense to generalize this type to `VirtQueueAffinity` since it could apply to network devices as well.

Reply:
Yep that makes sense - do we want to expand this configuration to be used for virtio-net devices? If I change the type name here it would probably make sense to add this as an option with the net flag in the cli and plumb it through the infrastructure. However, before I do that extra work it may make sense to submit this PR and see what the maintainers say?